### ISSUE TITLE
On Chrome & Safari dosen't work animation and images showing too small and circular problem

### ISSUE THREAD

**Issue by umit86** [CSS-Tricks/MovingBoxes#62]
Hello, the plugin is great and very usefull.

You can see the live example of my issues on here: http://www.umitkaraosmanoglu.com/works/yeni/index.html

This example works great on firefox, IE (all) and opera but on chrome and safari dosen't work animation and images showing too small.

And when I am circuling from last elemnt to first element dosen't work animaton.

Thanks, best regards,
Umit

**Comment by Mottie** [comment]
Hi Umit!

The issue with Chrome and Safari is that, I think, it's firing off the document ready before it even starts loading the images. So there are no set dimensions. So to solve this, set the image dimensions within the image itself:

``` html
<img src="myimage.jpg" width="400" height="300">
```

**Comment by umit86** [comment]
Hey Mottie,

I slove this small images problem just adding js codes in "document ready" and "window load" functions but the fisheye animation dosen't work!

http://www.umitkaraosmanoglu.com/works/yeni/index.html

**Comment by Mottie** [comment]
Hmm, I think most of the problems are due to the "mb-inner" div being absolutely positioned. It doesn't resize with the "mb-panel" so that's why it doesn't resize. When I have time, I'll have a closer look at how we can align the panels to the bottom instead of the top.

**Comment by umit86** [comment]
Yes, I set the <b>.mb-inside</b> div <code>position:absolute;</code> and <code>bottom:0;</code> for aligning images to bottom, it works all other browsers (including ie7, ie8, ie9) but on webkit not working which I didn't understand why!

Thanks a lot, for helping me.
Umit

**Comment by neeethumohan** [comment]
Hello,

In Mozilla and Internet explorer it didn't support for me .Images didn't moving.Please help how you resolve this issue

**Comment by Mottie** [comment]
Hi @neeethumohan!

The demo I'm sharing originally used jQuery edge, which breaks everything. Use jQuery v3.3.1 instead ([updated demo](http://jsfiddle.net/Mottie/acV4n/)).
